styles/index.scss

```diff
@@ -0,0 +1,170 @@
+:root {
+  --primary: #e71d26;
+  --primary-light: #e95b62;
+  --primary-dark: #ac030c;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+h2 {
+  margin: 2rem 0;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 2rem auto;
+}
+
+.header {
+  max-height: 100vh;
+
+  .cta {
+    display: flex;
+    margin-top: 5rem;
+    justify-content: center;
+    width: 100%;
+    height: 40rem;
+    &-img {
+      width: 80rem;
+      height: 40rem;
+      img {
+        max-width: 100%;
+        max-height: 100%;
+      }
+    }
+    &-text {
+      width: 100%;
+      margin-top: 4rem;
+      padding: 2rem 2rem 2rem 0;
+      line-height: normal;
+
+      span {
+        color: var(--primary);
+      }
+
+      p {
+        font-size: 1.2rem;
+        img {
+          width: 5rem;
+          margin-bottom: -1.3rem;
+        }
+      }
+    }
+    &-buttons {
+      margin-top: 3rem;
+      button {
+        font-size: 1.5rem;
+      }
+    }
+  }
+}
+
+.nav {
+  &__search {
+    position: relative;
+    input {
+      width: 300px;
+      border: none;
+      padding: 0.8rem 1rem;
+      border: 1px solid var(--gray-3);
+      border-radius: 0.4rem;
+
+      &:focus {
+        border: 1px solid transparent;
+        outline: 2px solid var(--primary);
+      }
+    }
+    button {
+      background: none;
+      border: none;
+      padding: 0;
+      position: absolute;
+      font-size: 1.2rem;
+      right: 1rem;
+      top: 0.5rem;
+    }
+  }
+  &__links {
+    .nav__link.active {
+      color: var(--primary);
+    }
+  }
+}
+
+.categories {
+  .category {
+    cursor: pointer;
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 0.5rem;
+    overflow: hidden;
+    span {
+      position: absolute;
+      color: var(--white);
+      font-size: 1.5rem;
+      text-align: center;
+      opacity: 0;
+      transition: all 0.3s ease-in-out;
+    }
+    img {
+      z-index: 5;
+      transition: all 0.5s;
+      width: 100%;
+      display: block;
+      height: auto;
+      transform: scale(1.5);
+    }
+    &:hover {
+      span {
+        position: absolute;
+        z-index: 10;
+        opacity: 1;
+      }
+      img {
+        filter: brightness(0.6);
+        transform: scale(2);
+      }
+    }
+  }
+}
+
+.footer {
+  margin: 1.3rem 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 10rem;
+  span {
+    font-size: 1rem;
+    display: inline-flex;
+    align-items: center;
+    margin-bottom: 1.5rem;
+    i {
+      font-size: inherit;
+      color: var(--primary);
+      margin: 0 0.4rem;
+    }
+    a {
+      margin-left: 0.3rem;
+    }
+  }
+  &__links i {
+    font-size: 1.5rem;
+    margin: 0 0.8rem;
+    color: #b8b8b8;
```

Comment:
Is this color used at only place? as you are defining color variables for other colors. So to keep consistency, you can declare a variable for this color too.